src/ILCompiler/src/Program.cs

```diff
@@ -280,7 +280,7 @@ private int Run(string[] args)
 
             // TODO: compiler switch for SIMD support?
             var simdVectorLength = (_isCppCodegen || _isWasmCodegen) ? SimdVectorLength.None : SimdVectorLength.Vector128Bit;
-            var targetDetails = new TargetDetails(_targetArchitecture, _targetOS, TargetAbi.CoreRT, simdVectorLength);
+            var targetDetails = new TargetDetails(_targetArchitecture, _targetOS, _isReadyToRunCodeGen ? TargetAbi.Jit : TargetAbi.CoreRT, simdVectorLength);
```

Comment:
Why do we need this? The only place `TargetAbi.Jit` is used is to encode EH info tables for the CoreRT JIT prototype. Does CoreCLR use the same EH info format as the CoreRT JIT prototype?

Reply:
I was being indecisive over whether I wanted to use a different Abi type to differentiate CoreRT / CoreRT r2r code gen. I'll clean this up. I was experimenting with a single ReadyToRunHeaderNode that could write the header for both but switching on Abi. It didn't look nice so I split them.